src/server/scripts/EasternKingdoms/zone_elwynn_forest.cpp

```diff
@@ -1,5 +1,5 @@
 /*
-* This file is part of the Pandaria 5.4.8 Project. See THANKS file for Copyright information
+* Copyright (C) 2022 MoPCore Reforged
```

Comment:
Really? No

Reply:
I copied/pasted from my file because I am also working on it on my side. 😉 It's simply the name I give to "my projects".

Reply:
And where you getall these numbers, you hahve sniffs or is it thinking values...

Reply:
I feel you are nervous. If you're in a bad mood, take a cup of tea. 💛🍵

Reply:
Makes no sense, we hahve enougth trash cores, not 1 more, have sniffs and use them or let someone with sniffs do that, we dont need more of these (thinked) values!